desktop/apps/auction2/components/page/Footer.js

```diff
@@ -2,14 +2,19 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import block from 'bem-cn'
 import renderTemplate from 'desktop/components/react/utils/render_template'
+import { connect } from 'react-redux'
 import { first } from 'underscore'
 
-export default function Footer (props) {
-  const { articles, auction, footerItems } = props
-  const footerItem = first(footerItems)
-  const showArticles = Boolean(articles.length)
-  const showFooterItems = footerItem && !auction.isAuctionPromo()
-  const b = block('auction-footer')
+function Footer (props) {
+  const {
+    articles,
+    footerItem,
+    showArticles,
+    showFooterItems,
+    sd
+  } = props
```

Comment:
👍 

I find it also keeps the interior of the component focused solely with markup generation for a very clean separation of concerns.